styles/go-plus-panel.less

```diff
@@ -106,4 +106,16 @@
     justify-content: center;
     text-align: center;
   }
+
+  .go-plus-panel-body {
+    font-family: Menlo, Consolas, 'DejaVu Sans Mono', monospace;
```

Comment:
any reason why we don't/can't reuse atom's font settings (fontFamily, fontSize, lineHeight) ?